### PR TITLE
fix typos

### PR DESCRIPTION
**Pull request by Bananeweizen**

### CODE REVIEW

File: spotbugs/etc/messages.xml [hunk 1]
```diff
@@ -633,7 +633,7 @@ about the validity or predictive ability of warnings, not as a way to find probl
   <Detector class="edu.umd.cs.findbugs.detect.DoInsideDoPrivileged">
     <Details>
 <![CDATA[
-<p> Look for code that should be executed inside doPrivileged blocks.</p>
+<p> Looks for code that should be executed inside doPrivileged blocks.</p>
```
[the remark this answers is not in the document]
Comment (Bananeweizen) [reply spotbugs/etc/messages.xml]: I don't think so. This is short for "This detector looks for..." and it is not an imperative, demanding the user to look for something. Just compare with all the other message above and below...

**Comment by Bananeweizen** [comment]
@iloveeclipse Andrey, please check again...

File: spotbugs/etc/messages.xml (hunk 1 above)
[the remark this answers is not in the document]
Comment (iloveeclipse) [reply spotbugs/etc/messages.xml]: Yes, I misunderstood the "detector" context here.

**Comment by Bananeweizen** [comment]
No problem, that happens to every one of us, every now and then. :) Thanks for merging.

**Pull request merged**